<launch>

  <arg name="worldName" default="garage"/>
  <arg name="robotNames" default="[robot_1, robot_2]"/>
  <arg name="namespace" default=""/>

  <node pkg="visualization_tools" exec="multiAgentVisualizationTools" name="multiAgentVisualizationTools" namespace="$(var namespace)" output="screen">
    <param name="exploredVoxelSize" value="0.3" />
    <param name="robotNames" value="$(var robotNames)" />
  </node>

</launch>
